system/Helpers/cookie_helper.php

```diff
@@ -0,0 +1,155 @@
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ *
+ * @package    CodeIgniter
+ * @author     CodeIgniter Dev Team
+ * @copyright  Copyright (c) 2014 - 2016, British Columbia Institute of Technology (http://bcit.ca/)
+ * @license    http://opensource.org/licenses/MIT    MIT License
+ * @link       http://codeigniter.com
+ * @since      Version 3.0.0
+ * @filesource
+ */
+
+// --------------------------------------------------------------------
+
+/**
+ * CodeIgniter Cookie Helpers
+ *
+ * @package     CodeIgniter
+ * @subpackage  Helpers
+ * @category    Helpers
+ * @author      CodeIgniter Dev Team
+ * @link        http://codeigniter.com/user_guide/helpers/cookie_helper.html
+ */
+
+if ( ! function_exists('set_cookie'))
+{
+
+    /**
+     * Set cookie
+     *
+     * Accepts seven parameters, or you can submit an associative
+     * array in the first parameter containing all the values.
+     *
+     * @param   mixed   $name
+     * @param   string  $value    The value of the cookie
+     * @param   string  $expire   The number of seconds until expiration
+     * @param   string  $domain   For site-wide cookie. 
+     *                            Usually: .yourdomain.com
+     * @param   string  $path     The cookie path
+     * @param   string  $prefix   The cookie prefix
+     * @param   bool    $secure   true makes the cookie secure
+     * @param   bool    $httpOnly true makes the cookie accessible via 
+     *                            http(s) only (no javascript)
+     * @see     (\Config\Services::response())->setCookie()
+     * @see     \CodeIgniter\HTTP\Response::setCookie()
+     * @return  void
+     */
+    function set_cookie
+    (
+        $name, 
+        string $value    = '', 
+        string $expire   = '', 
+        string $domain   = '', 
+        string $path     = '/', 
+        string $prefix   = '', 
+        bool   $secure   = false, 
+        bool   $httpOnly = false
+    )
+    {
+        (\Config\Services::response(new \Config\App()))->setcookie
```

Comment:
No need to send a new config file in here, since `Services::response` will return the existing IncomingResponse class. No need to instantiate a new class that won't be used.